examples/visualization/plot_contour.ipynb

```diff
@@ -64,7 +64,7 @@
     "\n",
     "    layers = []\n",
     "    for i in range(n_layers):\n",
-    "        n_units = int(trial.suggest_loguniform('n_units_l{}'.format(i), 32, 256))\n",
+    "        n_units = trial.suggest_int('n_units_l{}'.format(i), 32, 256)\n",
```

Comment:
This is just a comment. `suggest_loguniform` was used to search the order of magnitude of `n_units` similarly to the [Chainer example](https://github.com/pfnet/optuna/blob/master/examples/chainer_simple.py#L45).

I would like to accept this change because of the following two reasons:
1. The search space is small enough (it has about two hundred choices), and
1. the objective functions in visualization examples are almost the same, and it can be merged into one. In that case, the parallel coordinate plot does not support the log-scale axis (see https://github.com/pfnet/optuna/issues/669).

Reply:
Understood. Thanks for the explanation,